crates/wash-lib/src/generate/favorites.toml

```diff
@@ -21,8 +21,8 @@ subfolder = "examples/rust/components/http-hello-world"
 [[component]]
 name = "hello-world-tinygo"
 description = "a hello-world component (in TinyGo) that responds over an HTTP connection"
-git = "wasmCloud/wasmCloud"
-subfolder = "examples/golang/components/http-hello-world"
+git = "wasmCloud/go"
+subfolder = "templates/component/http-hello-world"
```

Comment:
I'd prefer not to have one of the templates be in a separate repo. Is there no way to fix this issue while keeping the Go template with the others?

Reply:
afaik we are restructuring so all go examples/templates will be in the `go` repo.

Reply:
Yeah your concern makes a lot of sense @asherillan , we may want to consider bringing back the project-templates repository to have a single place for these